backend/scheduler/metric_scheduler.py

```
@@ -16,12 +16,17 @@
 from backend.core.ai.sql_executor import SQLExecutor
 from backend.services.table_anomaly_service import TableAnomalyService
+from backend.config.settings import get_settings
+
+settings = get_settings()
 
 
 class MetricScheduler:
     """지표 스케줄러 (SQL 캐싱 지원)"""
     
-    def __init__(self, interval_minutes: int = 5):
+    def __init__(self, interval_minutes: int = None):
```

Ruff (0.13.1)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)